app/config/knights_config.py

```diff
@@ -0,0 +1,86 @@
+KNIGHTS = {
+    "lancelot": {
+        "name": "Lancelot",
+        "power": 35,
+        "hp": 100,
+        "armour": [],
+        "weapon": {
+            "name": "Metal Sword",
+            "power": 50
+        },
```

Comment:
There is a trailing comma after the 'weapon' dictionary. Removing this will maintain proper JSON-like formatting.